pkgs/applications/networking/browsers/nyxt/default.nix

```diff
@@ -1,56 +1,104 @@
-{ stdenv, lib, sbclPackages
-, makeWrapper, wrapGAppsHook3, gst_all_1
-, glib, gdk-pixbuf, cairo
-, mailcap, pango, gtk3
-, glib-networking, gsettings-desktop-schemas
-, xclip, wl-clipboard, notify-osd, enchant
+{ stdenv
+, lib
+, testers
+, wrapGAppsHook3
+, fetchzip
+, sbcl_2_4_6
```

Comment:
I think the convention for other packages is to just accept the bare program name, and pin it explicitly in all-packages.nix, though I may be mistaking